lessons/locales/fa-persian/access/modifying-permissions.md

```diff
@@ -1,56 +1,60 @@
-# Modifying Permissions
+# دستکاری مجوزها
 
-## Lesson Content
+## محتویات درس
 
-Changing permissions can easily be done with the <b>chmod</b> command. 
+شما با استفاده از فرمان <b>chmod</b> به راحتی می‌توانید مجوزها را تغییر دهید.
```

Comment:
Dude, markdown itself supports basic text styling. It's not necessary to use `html` markup. Just put ** or __ as usual tags.